media-sound/denemo/metadata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+	<email>waebbl@gmail.com</email>
+	<name>Bernd Waibel</name>
+</maintainer>
+<!-- TODO: Talk to sound project or remove it
+<maintainer type="project">
+	<email>sound@gentoo.org</email>
+	<name>Gentoo Sound project</name>
+</maintainer>
+-->
+<maintainer type="project">
+	<email>proxy-maint@gentoo.org</email>
+	<name>Proxy Maintainers</name>
+</maintainer>
+<longdescription>
+	Denemo is a music notation editor which uses
+	<pkg>media-sound/lilypond</pkg> to typeset music notation with LaTeX.
+</longdescription>
+<use>
+	<flag name="alsa">Enable alsa support through <pkg>media-libs/alsa-lib</pkg></flag>
+	<flag name="aubio" restrict="&gt;media-sound/denemo-2.2.0">
```

Comment:
Why all those `restrict`s? There's only one version to be added, isn't it?

Reply:
In case of USE flags, `restrict` is only necessary when you need to give different description to the same flag in different versions.

Reply:
Thanks for clarifying this. I removed them.